### ISSUE TITLE
Improvements needed for network routed

### ISSUE THREAD

**Issue by vbauzysvmware** [terraform-providers/terraform-provider-vcd#374]
* resource doesn't provide ability to provide description
* when pool address wrong, error isn't precise:
```
Error: error adding DHCP pool: error reconfiguring Edge Gateway: API Error: 400: [ 84284038-c2bd-41c0-86ff-3c608d003191 ] validation error on field 'configureGatewayServices.arg0.gatewayServiceConfigurationSpec.dhcpSpec.dhcpPools[0].lowIpAddress': String value has invalid format
```
* when Dns ip is wrong, error isn't precise:
```
Error: error: error creating the network: error instantiating a new OrgVDCNetwork: API Error: 400: [ afe4b21d-bee8-4e94-ac10-a228ca64cf32 ] validation error on field 'ipSettingList[0].nameServer1': String value has invalid format
```
* !!! when one of pool address is wrong - network is created, but don't exist in state file cause error is thrown.
* dhcp_pool field with details isn't refreshed when dhcp pool settings enabled/disabled or ip range is removed in UI. This can be reproduced: create route netw, goto actions "configure services" dhcp pool tab. !!! Overall another question risen here: should we allow to change edge gtw dhcp pool settings - as we would impact other networks which routed form same edge gtw?

**Comment by datacharmer** [comment]
Please add an example that reproduces the issue. What's a "wrong pool address" or "wrong Dns"?

**Comment by vbauzysvmware** [comment]
resource "vcd_network_routed" "TestAccVcdNetworkRoutedDhcp" {
  name         = "TestAccVcdNetworkRoutedDhcp"
  org          = "my1"
  vdc          = "vbVdc"
  edge_gateway = "vb_edge2"
  gateway      = "10.10.102.1111"
  dhcp_pool {
    start_address = "10.10.102.5111"
    end_address   = "10.10.102.100"
  }
}

**Comment by dataclouder** [comment]
Fixed in #402
